### DISCUSSION TITLE
Subfinder on termux

### DISCUSSION THREAD

**Discussion by Merss90** [projectdiscovery/subfinder#989] [Q&A] [answered]
Hi  I'm beginner. I have a problem while installing subfinder by termux on my phone. The message is:

go install -v github.com/projectdiscovery/subfinder/v2/cmd/subfinder@latest
github.com/projectdiscovery/utils/permission
# github.com/projectdiscovery/utils/permission
go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_other.go:10:6: checkCurrentUserRoot redeclared in this block
        go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_linux.go:16:6: other declaration of checkCurrentUserRoot
go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_other.go:15:6: checkCurrentUserCapNetRaw redeclared in this block
        go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_linux.go:21:6: other declaration of checkCurrentUserCapNetRaw

Can I have some help please.

**Comment by MetzinAround** [comment, marked as answer]
Hi @Merss90 , I check these errors in google and found a few answers to similar errors. 

https://stackoverflow.com/questions/62330084/fail-to-build-with-code-go-redeclared-in-this-block
https://github.com/golang/go/issues/46936
https://www.reddit.com/r/golang/comments/15ri3du/vs_code_go_main_redeclared_in_this_block/

Based on these similar errors, it might be an issue with your go version, looks like there's four versions installed? (10.6, 16.6, 15.6, 21.6) 

Might need to reinstall go or separate your workspaces if you need all those different versions.
